### DISCUSSION TITLE
Can I change the custom theme placeholder that I made with the theme maker?

### DISCUSSION THREAD

**Discussion by Arman-Espiar** [saadeghi/daisyui#3211] [Q&A] [answered]
How can I change the color of the placeholder of the custom theme I made with the theme builder? Apparently, it is not possible to change the color of this section.

**Comment by saadeghi** [comment]
What's theme placeholder?

**Reply by Arman-Espiar** [reply]
I meant [this part of the site](https://daisyui.com/theme-generator/). The appearance of the color of my theme was the same color as the color of the placheholder, that's why it is not shown, but the problem was solved by changing the color of the theme. The problem is that when you set the base-100 color to #a1a1a1, the placeholders are hidden.

**Reply by saadeghi** [reply]
I still don't know what you mean by "placeholder" 😅😬

**Reply by Arman-Espiar** [reply]
input placeholder ex : `<input class="w-full outline-none focus:outline-none" type="search" name="" id="" placeholder="@video #dotnet" />`

**Reply by saadeghi** [reply, marked as answer]
daisyUI doesn't apply any styles for that. It's the default browser color. But you can use Tailwind CSS `placehoder-*` class to change the color. For example `placeholder-primary`
